dev-libs/sway/sway-1.0_beta1.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+if [[ ${PV} == 9999 ]] ; then
+		EGIT_REPO_URI="https://github.com/swaywm/sway.git"
+		inherit git-r3
+else
+		# Version format: major.minor-beta.betanum
+		SWAY_PV="$(ver_cut 1-2)-$(ver_cut 3).$(ver_cut 4)"
+		SRC_URI="https://github.com/swaywm/sway/archive/${SWAY_PV}.tar.gz -> ${P}.tar.gz"
+		S="${WORKDIR}/sway-${SWAY_PV}"
+		KEYWORDS="~amd64 ~x86"
+fi
+
+inherit eutils fcaps meson pam
+
+DESCRIPTION="i3-compatible Wayland window manager"
+HOMEPAGE="https://swaywm.org"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="elogind fish-completion +swaybar +swaybg +swaylock systemd +tray wallpapers X zsh-completion"
+REQUIRED_USE="?? ( elogind systemd )"
+
+RDEPEND="~dev-libs/wlroots-0.1[systemd=,elogind=,X=]
+	>=dev-libs/json-c-0.13:0=
+	>=dev-libs/libinput-1.6.0:0=
+	dev-libs/libpcre
+	dev-libs/wayland
+	>=dev-libs/wayland-protocols-1.14
+	x11-libs/cairo
+	x11-libs/libxkbcommon
+	x11-libs/pango
+	x11-libs/pixman
+	elogind? ( >=sys-auth/elogind-237 )
+	swaybar? ( x11-libs/gdk-pixbuf:2[jpeg] )
+	swaybg? ( x11-libs/gdk-pixbuf:2[jpeg] )
+	swaylock? (
+		virtual/pam
+		x11-libs/gdk-pixbuf:2[jpeg]
+	)
+	systemd? ( >=sys-apps/systemd-237 )
+	tray? ( >=sys-apps/dbus-1.10 )
+	X? ( x11-libs/libxcb:0=[xkb] )"
+DEPEND="${RDEPEND}"
+BDEPEND="app-text/scdoc
+	virtual/pkgconfig"
+
+FILECAPS=( cap_sys_admin usr/bin/sway )
+
+src_prepare() {
+	eapply_user
```

Comment:
I'd just make this `default` in case at a later time we need to add a `PATCHES=()` array
we won't have to change this.

Reply:
Done.